.fixtures.yml

```diff
@@ -1,13 +1,9 @@
 fixtures:
   repositories:
     "stdlib":
-      "repo": "git://github.com/puppetlabs/puppetlabs-stdlib.git"
-      "ref": "master"
+      repo: "https://github.com/puppetlabs/puppetlabs-stdlib.git"
     "concat":
-      "repo": "git://github.com/puppetlabs/puppetlabs-concat.git"
-      "ref": "master"
+      repo: "https://github.com/puppetlabs/puppetlabs-concat.git"
     "hocon":
-      "repo": "git://github.com/puppetlabs/puppetlabs-hocon.git"
-      "ref": "master"
-  symlinks:
-    "puppet_authorization": "#{source_dir}"
+      repo: "https://github.com/voxpupuli/puppet-hocon.git"
```

Comment:
this will probably bite us later on. maybe we should first merge pr hocon pr and then clean this up?

Reply:
@bastelfreak https://github.com/voxpupuli/puppet-hocon/pull/1 should be ready for merge now :+1: